### DISCUSSION TITLE
Log files for SQLAlchemy

### DISCUSSION THREAD

**Discussion by LonareAman** [sqlalchemy/sqlalchemy#7548] [Usage Questions]
Where can I find all the historical transactions that are done by an application using SQLAlchemy? Is there a transactional log text file in which a user can see all their transactions?

**Comment by CaselIT** [comment]
Hi,

SQLAchemy does not save a transaction log. It does use the logging module so you may configure it so that a logging file is saved. See https://docs.sqlalchemy.org/en/14/core/engines.html?#configuring-logging for details
